benches/lib.rs

```diff
@@ -129,6 +129,31 @@ fn union_with(c: &mut Criterion) {
     });
 }
 
+fn union_of(c: &mut Criterion) {
```

Comment:
It seems that it measures the time it takes to aggregate a few deterministic tiny sets...  Tiny sets in a branchy setting lead to incorrect benchmarks because of branch prediction... see https://www.infoq.com/articles/making-code-faster-taming-branches/

I suggest using realistic data sets.  See for example https://github.com/RoaringBitmap/RoaringBitmap/tree/master/real-roaring-dataset/src/main/resources/real-roaring-dataset